### DISCUSSION TITLE
Welcome to CQUPT-ClassGrabbing Discussions!

### DISCUSSION THREAD

**Discussion by L2ncE** [LgoLgo/cqupt-grabber#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by L2ncE** [comment]
不再维护

**Comment by Rinai-R** [comment]
重启维护，有问题可以找我😎

**Reply by Rinai-R** [reply]
不再维护
